### DISCUSSION TITLE
Lighthouse offset value

### DISCUSSION THREAD

**Discussion by apark2459** [bitcraze/discussions#1486] [Q&A]
I had a question about the offset values for the lightsweeps. I am getting zero offset values for 3 of the sensors out of the 4. Is this expected? My understanding was that all 4 of the sensors should have the same offset values since it's detecting the same lightsweep. Also, does the offset value just represent the time that the revolution of the drum started? I would appreciate if I could get some help!

<img width="1021" alt="Screenshot 2024-07-24 at 10 42 51 PM" src="https://github.com/user-attachments/assets/8ff3e34a-7000-451f-addf-2b77d579132d">

**Comment by knmcguire** [comment]
Hi!

We will need to wait for more people to come back to the office in order for people to get help, but it seems that you are trying to interface with the lighthouse deck directly? Not with a Crazyflie in between right?

**Reply by apark2459** [reply]
Hello, Yes! I am trying to interface it with a Raspberry Pi (as part of a robot).

**Reply by apark2459** [reply]
I am currently working on estimating the geometry (i.e. pose) of the basestation but based on the workshop video, it looks like I need the angle measurements for all 4 IR sensors on the LH deck to do the PnP computation. So I was just wondering what the best approach would be since I seem to be unable to get 4 angle readings simultaneously from all 4 sensors!

**Reply by knmcguire** [reply]
Interfacing directly with the lighthouse deck is going to be really tough so currently there is no one at the office to help you with that (due to the summer holidays). You could consider putting the deck on a barebone crazyflie 2.1 or a Bolt on the deck and interface with that, that would be much easier.

**Reply by apark2459** [reply]
Yes, I have been trying and it seems like a very difficult task...may I ask if there is an option to purchase just the barebone crazyflie? Thanks!

**Reply by knmcguire** [reply]
Yes there is: https://store.bitcraze.io/products/crazyflie-2-1-bare-pcb

Buying a bolt would also be suitable for that https://store.bitcraze.io/collections/kits/products/crazyflie-bolt-1-1

The barebone pcb doesn't come with male pins so make sure to buy those as well: https://store.bitcraze.io/collections/spare-parts/products/male-deck-connector
